mingw-w64-git/git-wrapper.c

```diff
@@ -658,8 +664,13 @@ int main(void)
 	LPWSTR working_directory = NULL;
 
 	/* Determine MSys2-based Git path. */
+	#if defined(IS_ARM64) && IS_ARM64 == true
+	swprintf(msystem_bin, sizeof(msystem_bin),
+		L"arm64\\bin");
```

Comment:
This doesn't seem to work correctly yet (`mingw32` from above is added though):

![image](https://user-images.githubusercontent.com/17739158/103210855-22a88400-4907-11eb-8e98-edcd314afefc.png)

Reply:
This actually needs https://github.com/git-for-windows/git-sdk-32/pull/6. When I apply that PR, it works correctly: